### PR TITLE
Add 3 branch tests.

### PR DESCRIPTION
**Pull request by jrbenny35**
```test_add_branch```: Test adding a new branch.
```test_remove_branch```: Test removing a branch.
```test_duplicate_branch_name```: Test adding a branch with the same name as the control branch.

### CODE REVIEW

File: tests/integration/conftest.py [hunk 1]
```diff
@@ -13,17 +13,18 @@ def capabilities(capabilities):
 @pytest.fixture
 def firefox_options(firefox_options):
     """Set Firefox Options."""
-    firefox_options.headless = True
+    # firefox_options.headless = True
```
Comment (jaredkerim) [inline tests/integration/conftest.py]: Why is this commented out?

File: tests/integration/conftest.py [hunk 2]
```diff
@@ -13,17 +13,18 @@ def capabilities(capabilities):
...
     return firefox_options
 
 
-@pytest.fixture(scope='session', autouse=True)
+@pytest.fixture(scope="session", autouse=True)
```
Comment (jaredkerim) [inline tests/integration/conftest.py]: Oh interesting, I'm curious about this.  What does this do?

File: tests/integration/conftest.py (hunk 1 above)
Comment (jrbenny35) [reply tests/integration/conftest.py]: Good catch

File: tests/integration/conftest.py (hunk 2 above)
Comment (jrbenny35) [reply tests/integration/conftest.py]: This is so this fixture only runs once per test session. And ```autouse``` means it is used automatically. So this fixture runs automatically once per test session.

Comment (jrbenny35) [reply tests/integration/conftest.py]: Unless you are talking about the quotes, then that is just a python black package change.

**Pull request merged**